docs/UsersGuide/source/ConfigNewPlatform.rst

```diff
@@ -185,6 +194,8 @@ Otherwise you may be able to use pip3 (the Python3 package manager; may need to
 
    pip3 install jinja2 pyyaml f90nml
 
+Running the graphics scripts in ``${WORKDIR}/ufs-srweather-app/regional_workflow/ush/Python`` will require the additional packages ``pygrib``, ``cartopy``, ``matplotlib``, ``scipy``, and ``pillow``. These can be installed in the same way as described above.
```

Comment:
I did not know about "pillow".  Does that also need to be added to the graphics chapter?